### ISSUE TITLE
Requesting updated jar with commons-io and snakeyaml latest

### ISSUE THREAD

**Issue by sunilnagavelli** [signalfx/signalfx-java-tracing#146]
#### Description
Team, we are in a very peculiar situation, we currently could not upgrade to OTEL collector due to multiple dependencies, almost all our production loads are using this jar, unfortunately, we were tagged with security issues for the commons-oi and snakeyaml vulnerabilities, and currently, the version is below

[dd-java-agent/agent-jmxfetch/agent-jmxfetch.gradle](https://github.com/signalfx/signalfx-java-tracing/blob/113c952dd180413d2d51ba97ff97cf13a61115ed/dd-java-agent/agent-jmxfetch/agent-jmxfetch.gradle)

```
compile group: 'commons-io', name: 'commons-io', version: '2.4'  ---> 2.11.0
compile group: 'org.yaml', name: 'snakeyaml', version: '1.27'    ---> 1.32
```
we need the latest version to be updated, we tried many ways to build this in the local and upload the jar, but we failed to do that, how do we build this local in MACOS? is there a way that you can provide us with the updated jar just with this vulnerability update? it would be so kind of you to do so

@breedx-splk @mateuszrzeszutek @theletterf, could you please help us with this situation, we are trying very very hard to build it in the local but it is not helping, we are literally tearing hair with frustration, it would be great if you can shed some light on this.

<img width="753" alt="image" src="https://user-images.githubusercontent.com/40353788/190661911-879530a7-4846-4073-b772-4f7a23a80d40.png">

<img width="765" alt="image" src="https://user-images.githubusercontent.com/40353788/190665224-3fabae0a-2dd8-448e-a780-0d40b82ef189.png">

**Comment by sunilnagavelli** [comment]
@breedx-splk @mateuszrzeszutek @theletterf, I have opened a pull request just with these 2 file updates, would you mind looking into this and providing me the latest jar, please!!!

**Comment by breedx-splk** [comment]
This is resolved in [v0.48.0-sfx9](https://github.com/signalfx/signalfx-java-tracing/releases/tag/v0.48.0-sfx9). Please reopen if this doesn't work for you. Thanks!
